File: Cultural events/events.py
#!/usr/bin/env python3

# input data
N, V = map(int, input().strip().split())
prices = list(map(int, input().strip().split()))
vouchers = list(map(int, input().strip().split()))

prices.sort()
vouchers.sort()
#print(prices, vouchers)
num = 0
count = 0
for x in range(V):
    #print(x)
    if prices[num] <= vouchers[x]:
        count += 1
        num += 1
        #print(count, num)
    

print(count)  # print the result

File: Cultural events/events.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
	
//	freopen("input.txt", "r", stdin);
//	freopen("output.txt", "w", stdout);
	
	int N, V;
	
	cin >> N >> V;
	
	vector<int> prices(N), vo(V);
	
	for (int i = 0; i < N; i++){
		cin >> prices[i];
	}
	for (int i = 0; i < V; i++){
		cin >> vo[i];
	}
	
	sort(prices.begin(), prices.end());
	sort(vo.begin(), vo.end());
	
	int count = 0, num = 0;
	
	for (int i = 0; i < V && num < N; i++){
		if (prices[num] <= vo[i]){
			count++;
			num++;
		}
	}
	cout << count << "\n";
}
